gtsfm/utils/verification.py

```diff
@@ -74,38 +79,83 @@ def essential_to_fundamental_matrix(
     return np.linalg.inv(camera_intrinsics_i2.K().T) @ i2Ei1.matrix() @ np.linalg.inv(camera_intrinsics_i1.K())
 
 
-def compute_epipolar_distances(
-    normalized_coords_i1: np.ndarray, normalized_coords_i2: np.ndarray, i2Ei1: EssentialMatrix
+def compute_epipolar_distances_sed(
+    coordinates_i1: np.ndarray, coordinates_i2: np.ndarray, i2Fi1: np.ndarray
 ) -> Optional[np.ndarray]:
-    """Compute symmetric point-line epipolar distances between normalized coordinates of correspondences.
+    """Compute symmetric point-line epipolar distance between corresponding coordinates in two images. The SED is the
+    geometric point-line distance and is an over-estimate of the gold-standard reprojection error. The over-estimate
+    can reject correspondences which are actually inliers.
+
+    References: 
+    - "Fathy et al., Fundamental Matrix Estimation: A Study of Error Criteria", https://arxiv.org/abs/1706.07886
+    - "Hartley, R.~I. et al. Multiple View Geometry in Computer Vision.. Cambridge University Press, Pg 288"
+
+    Algorithm:
+    - l2 = i2Fi1 @ x1
+    - l1 = x2 @ i2Fi1 = [a1, b1, c1]
+    - n1 = EuclideanNorm(Normal(l1)) = sqrt(a1^2 + b1^2)
+    - n2 = EuclideanNorm(Normal(l2))
+    - SED^2 = ( l1 @ x1 )^2 / n1^2 + ( l1 @ x1 )^2 / n2^2
```

Comment:
SED = abs( l1 @ x1 ) / n1^2 + abs( l1 @ x1 ) / n2^2

Reply:
SED = abs( l1 @ x1 ) * sqrt(1/n1^2 + 1/n2^2)

Reply:
this line would be useful to put in the sampson part: https://docs.opencv.org/3.4/d9/d0c/group__calib3d.html#gacbba2ee98258ca81d352a31faa15a021

Reply:
done